### ISSUE TITLE
Default value of `--dimension` flag overrides `dimension` loader param value

### ISSUE THREAD

**Issue by yousefmoazzam** [DiamondLightSource/httomo#169]
When the `--dimension` flag is not provided to the CLI, it defaults to a value of 1 https://github.com/DiamondLightSource/httomo/blob/eec482d3cbe9e295394a992dcc6cb8fbd2d18723/httomo/cli.py#L49-L55

In `run_tasks()`, this value from the CLI is passed along https://github.com/DiamondLightSource/httomo/blob/eec482d3cbe9e295394a992dcc6cb8fbd2d18723/httomo/task_runner.py#L84-L90

and is used to update the dictionary of parameters for the loader https://github.com/DiamondLightSource/httomo/blob/eec482d3cbe9e295394a992dcc6cb8fbd2d18723/httomo/task_runner.py#L131

This results in the `dimension` parameter of `standard_tomo` being overridden by the `--dimension` flag even when `--dimension` is _not explicitly_ provided a value.

In other words, the `dimension` parameter of `standard_tomo` from the YAML is silently overridden if `--dimension` is not provided.

This feels like a bug, and that the proper behaviour should be something like:
- if `--dimension` is provided to the CLI, then override the `dimension` parameter of the loader
- if `--dimension`is _not_ provided to the CLI, then use the `dimension` parameter of the loader

Perhaps taking a step back though, maybe this ability to choose which dimension to read the data in should be available through only one of these options, I'm not certain what the benefit is in allowing it to be specified in two different ways: CLI, and a loader parameter?

**Comment by yousefmoazzam** [comment]
I think we decided at some point that once the loader is able to load sinogram blocks, the "dimension" should be inferred automatically by httomo and the user shouldn't have to deal with this (see #298)?

I'll close this for now @dkazanc, but feel free to undo/chime in with any comments.
